BaseApp/COD3041.TXT

```diff
@@ -0,0 +1,156 @@
+OBJECT Codeunit 3041 DotNet_RegexOptions

```

Comment:
This isn't a .Net wrapper, according to the guidelines.

Check [cod3037 ](https://github.com/Microsoft/cal-open-library/blob/master/BaseApp/COD3037.TXT) for reference on how to implement a .Net wrapper for enums.

Reply:
Hi Maria

The codeunit 3037 doesn't implement enums as they are implemented in .NET.
For RegexOptions - in .NET - you can do this:
  RegexOptions.Compiled | RegexOptions.IgnoreCase
Because the way Regex is designed, the feature above is very frequently used (I, myslef, use these options with pipes).
That's why I have added a number of methods allowing you to COMBINE options, not only use one of them.
Please, tell me how to implement it in NAV in any different way I did - I will happily change the code.
Best regards
Jacob

Reply:
Hi Maria
Thank you for your suggestion. I'm amending it as we speak; I'll publish a new pull request as soon as I have it ready.